Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -100,6 +100,7 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"gg_kill_reward" "Kill Percentage Reward"
 		"gg_assist_reward" "Assist Percentage Reward"
 		"gg_execution_reward" "Execution Percentage Reward"
+		"gg_heal_attacker" "Heal killer"
```

Comment:
I'd prefer something like "Heal on kill" idk why tho, cant put into words

(also this will need proper translation eventually, but i've been told we might be automating this soon so idc for now)